sandworm/lib/web.mlx

```diff
@@ -0,0 +1,237 @@
+        <span id="install-cmd" class_="pr-2 font-mono">
+          "curl https://dune.ci.dev/install | bash"
+        </span>
+        <Icons.copy id="copy-cmd" class_="rounded-tr rounded-br cursor-pointer" />
+        <script>
+          (JSX.unsafe {|
+            document.addEventListener('DOMContentLoaded', () => {
+              const copyBtn = document.querySelector('#copy-cmd');
+              const installCmd = document.querySelector('#install-cmd');
+              copyBtn.addEventListener('click', () => {
+                navigator.clipboard.writeText(installCmd.innerText)
+                  console.log('Async: Copying to clipboard was successful!');
+                }, function(err) {
+                  console.error('Async: Could not copy text: ', err);
+                });
+              });
+            |})
+        </script>
+      </div>
+    </div>
+    <p>
+      "If you wish to install the binary distribution manually, please follow the "
+      <a  class_="text-[var(--accent)]" href="#manual-installation">"manual installation instructions."</a>
+    </p>
+    <h2 class_="text-2xl font-medium pt-4 pb-2"> "Getting Started" </h2>
+    <p>
+      "You can create and run your first hello-world program using Dune like this:"
+      <pre>
+"
+$ dune init proj hello_world
+$ cd hello_world
+$ dune pkg lock
+$ dune exec hello_world
+"
+      </pre>
+    </p>
+    <p> 
+    "For more information on Dune and Package Management, check the "
+    <a class_="text-[var(--accent)]" href="https://dune.readthedocs.io/en/latest/"> "latest dune docs." </a>
+    </p>
+  </section>
+
+
+let faq () =  
+  <section id="faq" class_="flex flex-col py-10 gap-2">
+    <h2 class_="text-3xl font-black pb-4"> "Frequently Asked Questions" </h2>
+  </section>
+
+let installation_target target =
+  let open Metadata in
+  let title = Target.to_human_readable_string target in
+  let desc = Target.to_description target in
+   <li class_="min-w-1/3 max-w-1/3 px-4 py-2 flex flex-col gap-4 items-center border-[2px] border-solid border-[var(--accent)] rounded"> 
+     <h4 class_="font-bold">(JSX.string title)</h4>
+     <p>(JSX.string desc)</p>
+     <div class_="w-full flex flex-row justify-between gap-4">
+      <a class_="text-[var(--accent)]">"Download"</a>
+      <a>"Cert"</a>
+     </div>
+   </li>
+
+let release_history ~data () =  
+  let open Metadata in
+  <section id="release-history" class_="flex flex-col py-10 gap-2 text-left">
+    <h2 class_="text-2xl font-medium pb-4"> "Release History" </h2>
+    <p>
+      "Here is a table of all the releases of the Dune Developer Preview so far."
+    </p>
+    <section class_="flex flex-col gap-4">
+    (
+      data
+      |> List.map (fun (bundle: Bundle.t) -> 
+          let date = 
+            let (y,m,d) = bundle.date in
+            Format.sprintf "nightly-%d-%d-%d" y m d
+          in
+
+          let files = 
+            bundle.targets
+      |> List.map (fun (target: Target.t) ->
```

Comment:
Could you change indentation for future readability (maybe we should introduce ocamlformat-mlx?)?